scripts/readmeFiles.js

```diff
@@ -209,7 +209,7 @@ shell.ls("packages").forEach(folder => {
 
       initAutoReadme({ path, content });
     } catch (e) {
-      console.warn(e);
+      console.warn(folder, e);
```

Comment:
Just a little improvement – some of the components are still not working properly, and this helps identify which ones.